specs/api.kuflow.com/v2022-10-08/openapi.yaml

```diff
@@ -2067,17 +2174,6 @@ components:
           description: When this model type was last updated.
           type: string
           format: date-time
-      required:
-        - objectType
-      discriminator:
-        propertyName: objectType
-        mapping:
-          "AUTHENTICATION": "#/components/schemas/Authentication"
-          "PROCESS": "#/components/schemas/Process"
-          "PROCESS_PAGE_ITEM": "#/components/schemas/ProcessPageItem"
-          "TASK": "#/components/schemas/Task"
-          "TASK_PAGE_ITEM": "#/components/schemas/TaskPageItem"
-          "WORKER": "#/components/schemas/Worker"
```

Comment:
Is not needed to mark as deprecated?

Reply:
Not, in this case only we need to remove the discriminator section and mark the "objectType" property as deprecated.